app/controllers/public_facing_controller.rb

```diff
@@ -12,7 +12,7 @@ class PublicFacingController < ApplicationController
   end
 
   rescue_from GdsApi::HTTPErrorResponse do |exception|
```

Comment:
Could you also do `rescue_from GdsApi::HTTPServerError` (the 5XX superclass) and not have the `if` statement?